<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditRWDeviceParamForm</class>
 <widget class="QWidget" name="EditRWDeviceParamForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>412</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="tableWidgetDeviceList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>311</width>
     <height>192</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>DeviceID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>File name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Parameter</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>381</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>Device file name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditDLLFileName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditParameter">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>381</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>Parameter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonAddLine">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>20</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonDelLine">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>70</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSelectFile">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>270</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSet">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>370</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Set</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>Device ID</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxDeviceID">
   <property name="geometry">
    <rect>
     <x>141</x>
     <y>210</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-1</number>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
